Sources/Conduit/Networking/Data/AutoPurgingURLDataCache.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  AutoPurgingURLDataCache.swift
+//  
+//
+//  Created by Anthony Lipscomb on 8/2/21.
+//
+
+import Foundation
+
+public struct AutoPurgingURLDataCache: URLDataCache {
+
+    private let cache: NSCache<NSString, NSData>
```

Comment:
It would be less of a cache and more of a storage at that point and just wasn't the intention here. I was extended the work you all have done to add the ability to return Data instead of filtering through a UIImage which might strip that information